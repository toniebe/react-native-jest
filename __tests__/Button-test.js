import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../src/components/ButtonCustom';

test('renders correctly', () => {
    const tree = renderer.create(<Button />).toJSON();
    expect(tree).toMatchSnapshot();
  });