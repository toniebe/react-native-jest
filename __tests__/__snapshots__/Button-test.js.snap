// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "blue",
      "borderWidth": 1,
      "justifyContent": "center",
      "opacity": 1,
      "paddingVertical": 20,
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
      }
    }
  >
    Button
  </Text>
</View>
`;
